scripts/addb-py/chronometry/addb2db.py

```diff
@@ -58,7 +58,7 @@
 import yaml
 import numpy
 import time
-from peewee import *
+from peewee import IntegerField, TextField, FloatField, SqliteDatabase, Model, chunked, OperationalError
```

Comment:
Why is this needed? Does not look like cleanup.

Reply:
The star import makes ```pyflakes``` tool to raise the specified warning. Importing each module individually fixes these warnings.